codec/src/main/java/io/netty/handler/codec/compression/Lz4FrameEncoder.java

```diff
@@ -341,4 +395,12 @@ public void handlerRemoved(ChannelHandlerContext ctx) throws Exception {
         super.handlerRemoved(ctx);
         cleanup();
     }
+
+    public final int blockSize() {
```

Comment:
can we just make these new accessors package private for now, or are they generally useful outside the context of unit tests?